juju/juju
# Introduce metering-url controller config option.

## Description of change

Remove need for patching, set controller config. Similar to juju/juju#8896, for metering functionality in Juju.

## QA steps

`juju controller-config metering-url="https://api.staging.jujucharms.com/omnibus/v3" charmstore-url="https://api.staging.jujucharms.com/omnibus/v3"`

Deploy metered charm, set SLAs, plans, verify meter status is set. SLA, plan, budget commands should use controller-in-context for resolving APIs. Use --debug with all romulus commands to ensure the configured metering-url is used.

## Documentation changes

None at this time. Initial use case is supporting internal QA of Juju's integration with these APIs.

## Bug reference

None.
